src/App.tsx

```diff
@@ -48,7 +49,8 @@ export const App: React.FC = () => {
       let data = await response.json();
       let cleanedData = cleanMarketsData(data.results);
       setMarkets(cleanedData);
-      getDetails(cleanedData);
+      await getDetails(cleanedData);
```

Comment:
This function call isn't async. Is await needed here?

[I believe this is only needed for the resolution of Promises which is happening in the fetch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function)

Reply:
It's added so that you wait for details to be in state before routing to the results url

Reply:
I just tried it without await and it has no effect.
![Screen Shot 2021-07-28 at 9 06 41 PM](https://user-images.githubusercontent.com/12686237/127415962-d5c9a212-58ea-47d9-850f-6308dfdaebe4.png)

Reply:
Would love to run through the async / await process when we meet up tomorrow  to make sure I fully understand its purpose

Reply:
I'd like to try it with throttled internet and see what the ux difference is since my mentor suggested I add it